filebeat/_meta/common.reference.p2.yml

```diff
@@ -1,22 +1,22 @@
-#=========================== Filebeat prospectors =============================
+#=========================== Filebeat inputs =============================
 
-# List of prospectors to fetch data.
-filebeat.prospectors:
-# Each - is a prospector. Most options can be set at the prospector level, so
-# you can use different prospectors for various configurations.
-# Below are the prospector specific configurations.
+# List of input to fetch data.
+filebeat.inputs:
+# Each - is an input. Most options can be set at the input level, so
+# you can use different inputs for various configurations.
+# Below are the input specific configurations.
 
 # Type of the files. Based on this the way the file is read is decided.
-# The different types cannot be mixed in one prospector
+# The different types cannot be mixed in one input
 #
 # Possible options are:
 # * log: Reads every line of the log file (default)
 # * stdin: Reads the standard in
 
-#------------------------------ Log prospector --------------------------------
+#------------------------------ Log input --------------------------------
```

Comment:
Should we change this to say "Input" instead of "Log input" given the various types we now support?